synthetics/test_schema.go

```diff
@@ -302,30 +309,33 @@ func makeTestHealthSettingsSchema(mode schemaMode) *schema.Schema {
 }
 
 func makeTestMonitoringSettingsSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaRequired(mode, map[string]*schema.Schema{
 		"activation_grace_period": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"activation_time_unit": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: optionalOnRead(mode),
+			Required: requiredOnCreate(mode),
 			Computed: computedOnRead(mode),
 		},
 		"activation_time_window": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: optionalOnRead(mode),
+			Required: requiredOnCreate(mode),
 			Computed: computedOnRead(mode),
 		},
 		"activation_times": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: optionalOnRead(mode),
+			Required: requiredOnCreate(mode),
 			Computed: computedOnRead(mode),
 		},
 		"notification_channels": {
 			Type:     schema.TypeList,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
It think it is required (and sadly also non-functional).

Reply:
Addressed: a2e73bcd0db6697b5392fdb69a572f74228bc594

Reply:
Done